src/gobble.py

```diff
@@ -93,10 +95,24 @@ def client_thread(routes: Set[str]):
 def process_events(client: sseclient.SSEClient, trips_state: TripsStateManager):
     for event in client.events():
         try:
-            if event.event != "update":
+            logger.info(f"[{datetime.now().isoformat()}] Recieved {event.event} event")
```

Comment:
This log is very very noisy, can we update the existing logs downstream to maybe just add their type